src/app/hooks/useGlean.ts

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useEffect } from "react";
+import Glean from "@mozilla/glean/web";
+import * as appEvents from "../../telemetry/generated/appEvents";
+
+// Custom hook that initializes Glean and returns the Glean objects required
+// to record data.
+export const useGlean = () => {
+  // Initialize Glean only on the first render
+  // of our custom hook.
+  useEffect(() => {
+    Glean.initialize("monitor-testing", true, {
```

Comment:
**Important**: depending on the outcome of [this slack conversation](https://mozilla.slack.com/archives/C05ET81EPPW/p1693564326713539?thread_ts=1692727504.942899&cid=C05ET81EPPW), we may need to change the upload enabled parameter, currently fixed to `true`, to something else.